types/felt.go

```diff
@@ -7,106 +7,147 @@ import (
 	"fmt"
 	"io"
 	"math/big"
+	"regexp"
 	"strconv"
 	"strings"
 )
 
+const (
+	FIELD_GEN   int    = 3
+	FIELD_PRIME string = "3618502788666131213697322783095070105623107215331596699973092056135872020481"
+)
+
+var (
+	MaxFelt   = StrToFelt(FIELD_PRIME)
+	utfRegexp = regexp.MustCompile(`\w+`)
```

Comment:
That is not clear to me what short string should support, yet:
 
- that cannot be utf strings because utf is a multi-bytes format for instance utf8 is variable size format that can include 32bit characters and it would not fit the max size in that case.
- if that was something other than ascii, it would have to define the charset too (e.g. latin9 or other) . So I suppose that is ascii
- next come the problem of non graphic characters, like backspace. It think the pattern for the text should be only graphic characters up to 31.
- @ or whitespace should (and are not supported with \w) 
- empty strings would be nil anyway

So I propose the regexp below with a different variable name even if that is not accurate either:

```suggestion
	asciiRegexp = regexp.MustCompile(`^([[:graph:]]|[[:space:]]){1,31}$`)
```

see https://go.dev/play/p/fcYvD7bAd7A my tests

Reply:
Agreed, the new regex is a good change